utils.js

```diff
@@ -2,7 +2,7 @@
  * @param {string} input - A color in hexadecimal syntax. e.g. #fa6d01
  * @returns { R: string, G: string, B: string}
  */
-const hexToRGB = input => {
+const hexToRGB = (input) => {
```

Comment:
This whole file changes is my `prettier` doing his thing... If you don't want them, happy to roll them back!

Reply:
Great. Thanks 👍